### PR TITLE
fix: issue 3713

### LINKED ISSUE #3713: 【服务网格】Learning Journey 中 流量治理的网关规则名称不标准
image

应该是 Gateway，不是 Gateway Rule

https://docs.daocloud.io/en/mspider/intro/index.html#product-advantages

### PR DESCRIPTION
**Pull request by samzong**
/close #3713

Signed-off-by: samzong.lu <samzong.lu@gmail.com>

### CODE REVIEW

File: docs/en/docs/mspider/intro/index.md [hunk 1]
```diff
@@ -118,16 +118,12 @@ flowchart TD
...
-    traffic -.-> virtual[Virtual Service<br>Destination Rule<br>Gateway Rule]
+    traffic -.-> virtual[Virtual Service<br>Destination Rule<br>Gateway]
     security -.-> peer[Peer Authentication<br>Request Authentication<br>Authorization Policy]
     sidecar -.-> sidecarm[Namespace Sidecar<br>Workload Sidecar<br>Traffic Passthrough]
     watch -.-> watch2[Traffic Monitor<br>Traffic Topology]
     upgrade -.-> upgrade1[Upgrade Istio<br>Upgrade Sidecar]
 
-    classDef plain fill:#ddd,stroke:#fff,stroke-width:1px,color:#000;
-    classDef k8s fill:#326ce5,stroke:#fff,stroke-width:1px,color:#fff;
-    classDef cluster fill:#fff,stroke:#bbb,stroke-width:1px,color:#326ce5;
```
Comment (windsonsea) [inline docs/en/docs/mspider/intro/index.md:129]: 这 3 行是控制颜色的，删除了会不会没颜色了

**Comment by windsonsea** [comment]
See [preview](https://deploy-preview-3720--daocloud-docs.netlify.app/mspider/intro/).

File: docs/en/docs/mspider/intro/index.md (hunk 1 above)
Comment (samzong) [reply docs/en/docs/mspider/intro/index.md:129]: ok, my bad.

**Review by windsonsea — APPROVED** [review]

File: docs/en/docs/mspider/intro/index.md (hunk 1 above)
Comment (windsonsea) [reply docs/en/docs/mspider/intro/index.md]: all right, reverted.

**Pull request merged**